client/src/components/User/DiskUsage/Management/StorageManager.vue

```diff
@@ -11,11 +11,18 @@ import type { CleanableItem, CleanupOperation, CleanupResult } from "./Cleanup/m
 import CleanupOperationSummary from "./Cleanup/CleanupOperationSummary.vue";
 import CleanupResultDialog from "./Cleanup/CleanupResultDialog.vue";
 import ReviewCleanupDialog from "./Cleanup/ReviewCleanupDialog.vue";
+import BreadcrumbHeading from "@/components/Common/BreadcrumbHeading.vue";
 
 interface ModalDialog {
     openModal: () => void;
 }
 
+const breadcrumbItems = [
+    { title: "User Preferences", to: "/user" },
```

Comment:
Even if one of the ways to access the Storage Dashboard is through the user preferences, it is not really tied to the User preferences. So I would suggest removing the User Preferences as root.

```suggestion
```

Reply:
I included this because the user preferences page displays the user's storage usage and limit and directs them to the storage dashboard and I thought it's part of this section.

Reply:
This is really nice - I like the breadcrumbs overall. I have to say my initial gut impression was that rooting the storage dashboard in user preferences made no sense to me though - my eye immediately caught on that. I had the same thought as @davelopez before I even saw his comment.

Reply:
Thank you @davelopez and @jmchilton for your feedback. I removed them.

Reply:
I updated the screenshots.